Srcs/heiftojpeg/heiftojpeg.cpp

```diff
@@ -0,0 +1,183 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "hevcimagefilereader.hpp"
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <Magick++.h>
+#include <list>
+
+using namespace std;
+
+static void decodeData(ImageFileReaderInterface::DataVector data, Magick::Image *image)
+{
+    std::string hevcFileName = "tmp.hevc";
+    std::string bmpFileName = "tmp.bmp";
+    std::ofstream hevcFile(hevcFileName);
+    hevcFile.write((char*)&data[0],data.size());
+    hevcFile.close();
+    system(("ffmpeg -i tmp.hevc -loglevel panic -frames:v 1 -vsync vfr -q:v 1 -y -an " + bmpFileName).c_str());
+    *image = Magick::Image(bmpFileName);
+    remove(hevcFileName.c_str());
+    remove(bmpFileName.c_str());
+}
+
+static void addExif(ImageFileReaderInterface::DataVector exifData, std::string fileName)
+{
+    std::string exifFileName = "tmp.exif";
+    std::ofstream exifFile(exifFileName);
+    exifFile.write((char*)&exifData[0], exifData.size());
+    exifFile.close();
+    cout << "wrote exif to " << exifFileName << "\n";
```

Comment:
We should wrap all of these `cout <<` calls in a debug guard or something. Otherwise if we run this it will log a bunch of stuff into buffered streams and pollute the log.

Reply:
i added a `-v` command line option